src/js/view/controls/controlbar.js

```diff
@@ -568,5 +568,28 @@ define([
                 }
             }
         }
+
+        useInstreamTime(instreamModel) {
+            // While in instream mode, the time slider needs to move according to instream time
+            const timeSlider = this.elements.time;
+            if (!timeSlider) {
+                return;
+            }
+
+            instreamModel
+                .change('position', timeSlider.onPosition, timeSlider)
+                .change('duration', timeSlider.onDuration, timeSlider);
+        }
+
+        syncPlaybackTime(model) {
+            // When resuming playback mode, trigger a change so that the slider immediately resumes it's original position
+            if (!timeSlider) {
```

Comment:
Is this correct? You're checking if `timeSlider` is truthy before assigning a value to it.